modules/sc-mesh-secure-deployment/src/nats/comms_nats_controller.py

```diff
@@ -209,7 +99,8 @@ def __init__(
             str
         ] = self.__read_debug_config_from_file()
         self.__mesh_conf_request_processed = False
-        self.__comms_controller: CommsController = comms_controller
+        self.__comms_ctrl: comms_controller.CommsController = comms_ctrl
+        self.logger: logging = self.__comms_ctrl.logger.getChild("mdm_agent")
```

Comment:
Type annotation should probably be logging.Logger. Interestingly vscode still shows correct type for self.logger probably based on getChild() return type.

Reply:
SECO-3162